mk/spksrc.archs.mk

```diff
@@ -0,0 +1,36 @@
+# Common arch definitions
+
+# All available CPU architectures
+
+# Distinct SRM and DSM archs to allow handling of different TCVERSION ranges.
+# SRM - Synology Router Manager
+SRM_ARM7_ARCHS = northstarplus ipq806x dakota
+# DSM - all ARMv7 except SRM specific archs
+DSM_ARM7_ARCHS = alpine armada370 armada375 armada38x armadaxp comcerto2k monaco
```

Comment:
What is the use-case of theses two variables?
Could instead this be `SRM_ARCHS` and `DSM_ARCHS` more broadly?

Reply:
SRM_ARCHS are archs of Synology Routers and DSM_ARCHS are for Diskstations.
With the generic armv7 arch we have the problem that it is not possible to distinct the different TC-Version range (1.0-1.2 for SRM, 4.3 - 7.x for DSM)
Here I separated to be prepared for TCVERSION handing, but it has no priority for me in this PR.